src/Serilog.Sinks.Loki/LokiBatchFormatter.cs

```diff
@@ -69,6 +78,28 @@ public void Format(IEnumerable<LogEvent> logEvents, ITextFormatter formatter, Te
                     }
                 }
 
+                foreach (KeyValuePair<string, LogEventPropertyValue> property in logEvent.Properties)
+                {
+                    // Some enrichers pass strings with quotes surrounding the values inside the string,
+                    // which results in redundant quotes after serialization and a "bad request" response.
+                    // To avoid this, remove all quotes from the value.
+                    // We also remove any \r\n newlines and replace with \n new lines to prevent "bad request" responses
+                    // We also remove backslashes and replace with forward slashes, Loki doesn't like those either
+                    var propertyValue = property.Value.ToString().Replace("\"", "").Replace("\r\n", "\n").Replace("\\", "/");
+
+                    switch (DetermineHandleActionForProperty(property.Key))
+                    {
+                        case HandleAction.Discard:
+                            continue;
+                        case HandleAction.SendAsLabel:
+                            stream.Labels.Add(new LokiLabel(property.Key, propertyValue));
```

Comment:
Only in this case do `propertyValue = propertyValue.Replace("\"", "").Replace("\\", "/");`